server/src/main/java/org/elasticsearch/action/admin/cluster/stats/TransportClusterStatsAction.java

```diff
@@ -70,6 +85,8 @@ public class TransportClusterStatsAction extends TransportNodesAction<
     SubscribableListener<TransportClusterStatsAction.AdditionalStats>> {
 
     public static final ActionType<ClusterStatsResponse> TYPE = new ActionType<>("cluster:monitor/stats");
+
+    public static final FeatureFlag CCS_TELEMETRY_FEATURE_FLAG = new FeatureFlag("ccs_telemetry");
```

Comment:
I still think we shouldn't be using a feature flag for this feature.

Reply:
Do you mean just for the stats action?